arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/CreateSection.java

```diff
@@ -243,11 +250,14 @@ public void resolveUponContextEntry(Hub hub) {
     creatorAccountFragment.rlpAddrSubFragment(rlpAddrSubFragment);
 
     final AccountFragment createeAccountFragment =
-        accountFragmentFactory.make(
+        accountFragmentFactory.makeWithTrm(
```

Comment:
We trim the deployment address. Not because it could be too long, but because the first occurrence of any address requires trimming. And this may be it for the deployment address (it will be except for adversarial cases testing edge-cases.)